code/obj/item/fitness.dm

```diff
@@ -251,3 +251,42 @@
 		src.add_fingerprint(user)
 		spam_flag = world.time + 2 SECONDS
 	return
+
+/obj/item/screamingtunapillow
+	name = "Tuna Pillow"
+	desc = "It's a pillow that has a tuna on it."
+	icon = 'icons/obj/plushies.dmi'
+	icon_state = "tunapillow"
+	var/spam_flag = 0
+
+/obj/item/screamingtunapillow/attack_self(mob/user as mob) //copy and pasted from duck code
+	if (spam_flag < world.time)
+		if (ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if (H.sims)
+				H.sims.affectMotive("fun", 1)
+		spam_flag = 1
+		if (narrator_mode)
+			playsound(user, 'sound/vox/duct.ogg', 50, 1)
+		else
+			if (prob (50))
+				playsound(user, 'sound/effects/tunascream01.ogg', 50, 1)
+				user.visible_message("<B>THE TUNA PILLOW SCREAMS!</B>")
+			if (prob (50))
+				playsound(user, 'sound/effects/tunascream02.ogg', 50, 1)
+				user.visible_message("<B>THE TUNA PILLOW SCREAMS!</B>")
```

Comment:
this is a 25% chance to make no sound at all, and a 25% chance to make *both* screams almost simultaneously with two messages. Revise to 
`if(prob(50)) [sound 1] else [sound 2]`
and put the visible message outside the conditional.

Reply:
So far in testing, I managed to have only either both screams play (they layer on top of each other pretty well and I find it funny), or just one. How could I keep that effect while eliminating the no sound at all chance?

Reply:
please don't do that, it's bad for client playback and makes volume suck ass